CHANGELOG.md

```diff
@@ -17,6 +17,14 @@ The REST API versions in the table are just for your information as the values a
 
 - We ask developers to consider using the Headline component instead of the KPI component. The KPI component may be eventually marked as deprecated in one of the next major versions.
 
+## 7.6.1
+
+May 20, 2020
+
+### Fixed
+
+- Removed the hooks that could not work with React version 16.5.2
```

Comment:
It could be
`Remove using React Hooks that could not work with React version 16.5.2`